x-pack/plugins/cases/server/client/cases/get.test.ts

```diff
@@ -44,4 +44,13 @@ describe('get', () => {
       );
     });
   });
+
+  describe('getCategories', () => {
```

Comment:
Let's add some tests where we validate a) if excess fields are returned by the `caseService` they are stripped out b) if fields are missing in the response of the `caseService` an error is thrown and c) a happy path

Reply:
The thing is this service does not return fields, it just returns an array of strings. What I can do is make sure it throws an error if the response is not a `string[]`?